.yarnrc

```diff
@@ -1,3 +1,3 @@
 disturl "https://electronjs.org/headers"
-target "9.3.3"
+target "11.0.2"
```

Comment:
I think we are not yet having internal builds for this right?

Reply:
yup, will merge once the builds are available, can expect them in beginning of next week.